jolly-jellyfish/src/page_maker/templates/page_maker/user_detail.html

```diff
@@ -0,0 +1,25 @@
+{% extends "page_maker/base.html" %}
+
+{% block content %}
+<div>
+  username: {{ viewed_user.username }}
```

Comment:
```suggestion
{% block content %}
username: {{ viewed_user.username }}
<div>
```
As above, styling concerns.

Reply:
Shouldn't there be a header too?

Reply:
Oh, you're right
```suggestion
{% block content %}
<h2>Username: {{ viewed_user.username }}</h2>
<div>
```